app_dart/test/request_handlers/refresh_chromebot_status_test.dart

```diff
@@ -189,19 +189,19 @@ void main() {
             Task(key: commit.key.append(Task, id: 123), commitKey: commit.key, name: 'Linux A', status: Task.statusNew);
         config.db.values[commit.key] = commit;
         config.db.values[task.key] = task;
-        final Map<BranchLuciBuilder, Map<String, List<LuciTask>>> luciTasks =
-            Map<BranchLuciBuilder, Map<String, List<LuciTask>>>.fromIterable(builders,
-                key: (dynamic builder) => BranchLuciBuilder(luciBuilder: builder as LuciBuilder?, branch: 'master'),
-                value: (dynamic builder) => <String, List<LuciTask>>{
-                      'abc': <LuciTask>[
-                        const LuciTask(
-                            commitSha: 'abc',
-                            ref: 'refs/heads/master',
-                            status: Task.statusSucceeded,
-                            buildNumber: 1,
-                            builderName: 'abc')
-                      ],
-                    });
+        final Map<BranchLuciBuilder, Map<String, List<LuciTask>>> luciTasks = {
+          for (var builder in builders)
```

Comment:
@keyonghan I prefer this use of var (we don't have to explicitly list the type, but it's implicit from above). WDYT?

Reply:
Any reason why not specify the `LuciBuilder` type explicitly? https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic

Reply:
@XilaiZhang can you re-add back the variable types to the lint?

Reply:
yes sir